wpimath/src/main/native/include/frc/geometry/Pose2d.h

```diff
@@ -176,6 +179,20 @@ class WPILIB_DLLEXPORT Pose2d {
    */
   Twist2d Log(const Pose2d& end) const;
 
+  /**
+   * Returns the nearest Pose2d from a collection of poses
+   * @param poses The collection of poses.
+   * @return The nearest Pose2d from the collection.
+   */
+  Pose2d Nearest(std::span<Pose2d const> poses) const;
```

Comment:
Our other uses of std::span put the const first, so `std::span<const Pose2d>` in this case.